docs/docsite/rst/os_guide/windows_setup.rst

```diff
@@ -280,162 +235,139 @@ This will output something like:
+
+.. code-block:: powershell
+
     winrs -r:https://server:5986/wsman -u:Username -p:Password -ssl ipconfig
 
-    # Test out HTTPS, ignoring certificate verification
+The command will fail if the certificate is not verifiable.
+   
+* To test HTTPS ignoring certificate verification:
+
+.. code-block:: powershell
+
     $username = "Username"
     $password = ConvertTo-SecureString -String "Password" -AsPlainText -Force
     $cred = New-Object -TypeName System.Management.Automation.PSCredential -ArgumentList $username, $password
 
     $session_option = New-PSSessionOption -SkipCACheck -SkipCNCheck -SkipRevocationCheck
     Invoke-Command -ComputerName server -UseSSL -ScriptBlock { ipconfig } -Credential $cred -SessionOption $session_option
 
-If this fails, the issue is probably related to the WinRM setup. If it works, the issue may not be related to the WinRM setup; please continue reading for more troubleshooting suggestions.
+If any of the above commands fail, the issue is probably related to the WinRM setup.
 
 HTTP 401/Credentials Rejected
 +++++++++++++++++++++++++++++
-A HTTP 401 error indicates the authentication process failed during the initial
-connection. Some things to check for this are:
+An HTTP 401 error indicates the authentication process failed during the initial
+connection. You can check the following to troubleshoot:
 
-* Verify that the credentials are correct and set properly in your inventory with
-  ``ansible_user`` and ``ansible_password``
+* The credentials are correct and set properly in your inventory with the ``ansible_user`` and ``ansible_password`` variables.
 
-* Ensure that the user is a member of the local Administrators group or has been explicitly
-  granted access (a connection test with the ``winrs`` command can be used to
-  rule this out).
+* The user is a member of the local Administrators group, or has been explicitly granted access. You can perform a connection test with the ``winrs`` command to rule this out.
 
-* Make sure that the authentication option set by ``ansible_winrm_transport`` is enabled under
-  ``Service\Auth\*``
+* The authentication option set by the ``ansible_winrm_transport`` variable is enabled under ``Service\Auth\*``.
 
-* If running over HTTP and not HTTPS, use ``ntlm``, ``kerberos`` or ``credssp``
-  with ``ansible_winrm_message_encryption: auto`` to enable message encryption.
-  If using another authentication option or if the installed pywinrm version cannot be
-  upgraded, the ``Service\AllowUnencrypted`` can be set to ``true`` but this is
-  only recommended for troubleshooting
+* If running over HTTP and not HTTPS, use ``ntlm``, ``kerberos`` or ``credssp`` with the ``ansible_winrm_message_encryption: auto`` custom inventory variable to enable message encryption. If you use another authentication option, or if it is not possible to upgrade the installed ``pywinrm`` package, you can set ``Service\AllowUnencrypted`` to ``true``. This is recommended only for troubleshooting.
```

Comment:
use ``ntlm``, ``kerberos`` or ``credssp`` - these options are valid for `ansible_winrm_transport`, right?

because `ansible_winrm_message_encryption` variable has different options (`auto`, `laways`, `never`)